modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/L0TrustClient.scala

```diff
@@ -0,0 +1,36 @@
+package org.tessellation.sdk.http.p2p.clients
+
+import cats.effect.Async
+import cats.effect.std.Random
+import cats.syntax.flatMap._
+
+import org.tessellation.schema.trust.TrustScores
+import org.tessellation.sdk.domain.cluster.storage.L0ClusterStorage
+import org.tessellation.sdk.domain.trust.storage.OrdinalTrustMap
+import org.tessellation.sdk.http.p2p.PeerResponse
+import org.tessellation.sdk.infrastructure.snapshot.RandomPeerSelect
+import org.tessellation.security.SecurityProvider
+
+import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+import org.http4s.client.Client
+
+trait L0TrustClient[F[_]] {
+  def getLatestTrust: F[TrustScores]
+  def getLatestCurrentOrdinalTrust: F[OrdinalTrustMap]
+}
+
+object L0TrustClient {
+  def make[F[_]: Async: SecurityProvider: Random](client: Client[F], l0Storage: L0ClusterStorage[F]): L0TrustClient[F] =
+    new L0TrustClient[F] {
+
+      def getLatestTrust: F[TrustScores] =
+        l0Storage.getPeers
+          .flatMap(peers => RandomPeerSelect.make(peers.toSortedSet).select)
+          .flatMap(peer => PeerResponse[F, TrustScores]("trust/latest")(client).run(peer))
+
+      def getLatestCurrentOrdinalTrust: F[OrdinalTrustMap] =
+        l0Storage.getPeers
+          .flatMap(peers => RandomPeerSelect.make(peers.toSortedSet).select)
+          .flatMap(peer => PeerResponse[F, OrdinalTrustMap]("trust/current-ordinal/latest")(client).run(peer))
```

Comment:
P2P client should have no logic like getting peers from storage. It should be a simple data access object.
Peers should be passed as an argument.

Reply:
Per offline discussions, `/trust/latest` has been renamed `/trust/current` and `/trust/current-ordinal/latest` has been renamed `/trust/previous`. The methods in `L0TrustClient` have been renamed accordingly. Said methods now accept an `L0Peer` as a parameter and the `L0ClusterStorage` is no longer a parameter to the `make`. `RandomPeerSelect` has been removed since it is not currently being used.